cpukit/score/cpu/arm/include/rtems/score/cpu.h

```diff
@@ -183,8 +184,11 @@
 
 #define ARM_EXCEPTION_FRAME_VFP_CONTEXT_OFFSET 72
 
+#define ARM_STACK_PROT_ATTR_OFFSET  44
```

Comment:
Please make that `ARM_CONTEXT_STACK_PROT_ATTR_OFFSET` Also, there are a few #ifdefs that, if defined, will make this offset incorrect. You would want to to #ifdef it too